src/export-members.ts

```diff
@@ -1,40 +1,78 @@
 import snapshot from '@snapshot-labs/snapshot.js'
 import { Networks } from './entities/Networks'
 import { SnapshotSpace } from './interfaces/GovernanceProposal'
-import { STRATEGIES, Vote } from './interfaces/Members'
-import { fetchGraphQL, flattenArray, saveToCSV, saveToJSON, snapshotUrl, splitArray } from './utils'
+import { DelegationInfo, MemberInfo, STRATEGIES, Vote } from './interfaces/Members'
+import { fetchDelegations, fetchGraphQL, flattenArray, parseVP, saveToCSV, saveToJSON, snapshotUrl, splitArray } from './utils'
 
-const MAX_RETRIES = 20
-
-require('dotenv').config()
-
-export interface MemberInfo {
-  address: string
-  totalVP: number
-  manaVP: number
-  landVP: number
-  namesVP: number
-  delegatedVP: number
-}
+const MAX_RETRIES = 10
 
 const space = SnapshotSpace.DCL
 const network = Networks.getEth().id.toString()
-const blockNumber = 'latest'
+
+/*
+* Until Snapshot makes it possible to obtain the delegated VP in a more optimal way, 
```

Comment:
This can be replaced with the new vp distribution endpoint right?

Reply:
It doesn't work because it asks for the ID of a proposal and the script exports the current VP of each member

<img width="882" alt="image" src="https://user-images.githubusercontent.com/45410089/199352703-eb674fc3-aca0-4e29-941a-2b82df9406e5.png">